setup.cfg

```diff
@@ -1,5 +1,5 @@
 [flake8]
-ignore = E501,F403,E226,W503
+ignore = E501,F403,E226,E722,W503
```

Comment:
Should we also ignore error E731?
It makes the following error in another branch
"test_fieldset.py:186:5: E731 do not assign a lambda expression, use a def"

Reply:
I didn't realise that using lambda functions is against PEP8, but actually that does make sense. Why not use a `def` indeed, for readability, in PR #228?

Reply:
I'll do the change in the other branch.